rust/worker/src/execution/operators/projection.rs

```diff
@@ -84,446 +81,69 @@ impl ChromaError for ProjectionError {
-        assert!(id_to_data.contains_key(&String::from("embedding_id_3")),);
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_1"))
-                .as_ref()
-                .expect("Expected value")
-                .1
-                .as_ref()
-                .expect("Expected not none")
-                .get(&String::from("hello"))
-                .expect("Expected key to be present"),
-            &MetadataValue::Str(String::from("new_world"))
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_1"))
-                .as_ref()
-                .expect("Expected value")
-                .1
-                .as_ref()
-                .expect("Expected not none")
-                .get(&String::from("bye"))
-                .expect("Expected key to be present"),
-            &MetadataValue::Str(String::from("world"))
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_1"))
-                .as_ref()
-                .expect("Expected value")
-                .1
-                .as_ref()
-                .expect("Expected not none")
-                .get(&String::from("hello_again"))
-                .expect("Expected key to be present"),
-            &MetadataValue::Str(String::from("new_world"))
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_3"))
-                .as_ref()
-                .expect("Expected value")
-                .1
-                .as_ref()
-                .expect("Expected not none")
-                .get(&String::from("hello"))
-                .expect("Expected key to be present"),
-            &MetadataValue::Str(String::from("new_world"))
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_3"))
-                .as_ref()
-                .expect("Expected value")
-                .1
-                .as_ref()
-                .expect("Expected not none")
-                .get(&String::from("hello_again"))
-                .expect("Expected key to be present"),
-            &MetadataValue::Str(String::from("new_world"))
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_1"))
-                .as_ref()
-                .expect("Expected value")
-                .0
-                .as_ref()
-                .expect("Expected not none"),
-            &String::from("This is a document about cats.")
-        );
-        assert_eq!(
-            id_to_data
-                .get(&String::from("embedding_id_3"))
-                .as_ref()
-                .expect("Expected value")
-                .0
-                .as_ref()
-                .expect("Expected not none"),
-            &String::from("This is a document about dogs.")
-        );
```

Comment:
could we keep the tests?

Reply:
Tests will be reintroduced in a later PR.

Reply:
Update: tests are re-introduced.